server/go.mod

```diff
@@ -8,5 +8,10 @@ require (
 	github.com/jackc/pgx v3.6.2+incompatible
 	github.com/jackc/pgx/v4 v4.6.0 // indirect
 	github.com/stretchr/testify v1.5.1
+	github.com/yuin/goldmark v1.1.32 // indirect
```

Comment:
can you run `go mod tidy`? This finds any unused packages and removes them from the go.mod file

Reply:
Ran `go mod tidy`